rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -284,3 +287,37 @@ def rate_limit_write(bucket, max_write_ops, ssl=None):
     )
     stdout, stderr = run_subprocess(cmd)
     assert "503" in str(stderr), "Rate limit slowdown not observed, failing!"
+
+
+def test_full_sync_at_archive(bucket_name):
+    """
+    test_full_sync_at_archive zone for a bucket
+    """
+    zone_name = "archive"
+    bucket_name = f"tenant/{bucket_name}"
+    cmd_bucket_stats = f"radosgw-admin bucket stats --bucket {bucket_name}"
+    log.info(f"collect bucket stats for {bucket_name} at local site")
+    local_bucket_stats = json.loads(utils.exec_shell_cmd(cmd_bucket_stats))
+    local_num_objects = local_bucket_stats["usage"]["rgw.main"]["num_objects"]
+    local_size = local_bucket_stats["usage"]["rgw.main"]["size"]
+    log.info(f"remote zone is {zone_name}")
+    remote_ip = utils.get_rgw_ip_zone(zone_name)
+    remote_site_ssh_con = utils.connect_remote(remote_ip)
+    log.info("Restart the gateways at the archive site")
+    remote_site_ssh_con.exec_command("ceph orch restart rgw.shared.arc")
+    log.info(f"collect bucket stats for {bucket_name} at remote site {zone_name}")
+    log.info("Wait for the sync lease period of 1200 seconds")
+    time.sleep(1800)
```

Comment:
The time.sleep mentions 1800, could you fix the typo

Reply:
Yes it should be 1800 (1200 for lease period and another 10 minutes for data sync.)